### PR TITLE
refactor(sbb-teaser): use LinkProperties interface

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/components/sbb-teaser/sbb-teaser.tsx [hunk 1]
```diff
@@ -17,7 +24,28 @@ import { AccessibilityProperties } from '../../global/interfaces/accessibility-p
...
+   */
+  @Prop({ reflect: true }) public isStacked: boolean;
+
+  /**
+   * Heading level of the sbb-title element (e.g. h1-h6).
+   */
+  @Prop() public titleLevel: InterfaceTeaserAttributes['titleLevel'] = '5';
+
+  /** The href value you want to link to. */
+  @Prop() public href!: string | undefined;
```
Comment (DavideMininni-Fincons) [inline src/components/sbb-teaser/sbb-teaser.tsx:41]: Since it's mandatory, is the undefined type needed?

**Review by DavideMininni-Fincons — COMMENTED** [review]
Looks good, added just a comment;
should we change the event type to `Event` in src/global/interfaces/link-button-properties.ts line 76 (the `emitButton` function)?

File: src/components/sbb-teaser/sbb-teaser.tsx (hunk 1 above)
Comment (jeripeierSBB) [reply src/components/sbb-teaser/sbb-teaser.tsx:41]: I removed the mandatory flag, as it renders to span if there is no href

**Pull request merged**